bigtable/client/internal/unary_rpc_utils.h

```diff
@@ -161,22 +161,56 @@ struct UnaryRpcUtils {
       MemberFunction function,
       typename CheckSignature<MemberFunction>::RequestType const &request,
       char const *error_message) {
+    return CallWithRetryBorrow(client, *rpc_policy, *backoff_policy, function,
+                               request, error_message);
+  }
+
+  /**
+   * Call a simple unary RPC with retries borrowing the RPC policies.
+   *
+   * This implements `CallWithRetry()`, but does not assume ownership of the RPC
+   * policies.  Some RPCs, notably those with pagination, can reuse most of the
+   * code in `CallWithRetry()` but must reuse the same policies across several
+   * calls.
+   *
+   * @tparam MemberFunction the signature of the member function.
+   * @param client the object that holds the gRPC stub.
+   * @param rpc_policy the policy controlling what failures are retryable.
+   * @param backoff_policy the policy controlling how long to wait before
+   *     retrying.
+   * @param function the pointer to the member function to call.
+   * @param request an initialized request parameter for the RPC.
+   * @param error_message include this message in any exception or error log.
+   * @return the return parameter from the RPC.
+   * @throw std::exception with a description of the last RPC error.
+   */
+  template <typename MemberFunction>
+  // Disable the function if the provided member function does not match the
```

Comment:
This depends on the level of experience we expect from the developers of the project.  I am comfortable removing such comments, but Google developers tend to find templates obscure, so I err on being overly verbose.

I expect this will be used in a handful of places in the code, so I am Okay with the verbose comments for the time being.